### PR TITLE
September Changelog

### PR DESCRIPTION
**Pull request by loujaybee**
> **Note** Only make changes to this PR to add ad-hoc changelog content, the automated list will be re-generated. To update the changelog list of items, update the release notes section on your PR directly. To omit, simply add `NONE` to the release notes field. 

Script for re-generating: `node scripts/generate-changelog.js 2022-09-30 2022-09-01 2022-09-30`

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2760"></a>

**Features to highlight:**
- [x] [New ports view for VS Code Browser](https://www.gitpod.io/changelog/new-ports-explorer-changelog-view-and-connection-improvements-for-vs-code#new-ports-explorer-in-vs-code-browser)
- [x] [JetBrains common actions](https://github.com/gitpod-io/gitpod/issues/12884)
- [x] JetBrains workspace class information
- [x] JetBrains reconnecting windows
- [x] Alt text for images added
- [x] Copy images to current repo (not external)
- [x] SSH Gateway default for VS Code Desktop users in Self-Hosted
- [x] @jacobparis update header image

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2760"></a>

### CODE REVIEW

File: src/lib/contents/changelog/2022-09-30.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
+- [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
+- [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
+- [#12683](https://github.com/gitpod-io/gitpod/pull/12683) - [installer]: config detects shiftfs and containerd values <Contributors usernames="MrSimonEmms,nandajavarma,sagor999" />
+- [#12679](https://github.com/gitpod-io/gitpod/pull/12679) - [installer]: create config command and deprecate init command <Contributors usernames="ArthurSens,MrSimonEmms,mads-hartmann,nandajavarma" />
+- [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
```
Comment (filiptronicek) [inline src/lib/contents/changelog/2022-09-30.md:32]:
```suggestion
- [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />
- [#13203](https://github.com/gitpod-io/gitpod/pull/13203) - [server]: Add a liveness probe which fails when the nodejs event loop lag exceeds a certain threshold <Contributors usernames="andrew-farries,easyCZ" />
- [#13080](https://github.com/gitpod-io/gitpod/pull/13080) - JetBrains Gateway: Avoid 30 seconds delay when connecting to a workspace using an expired token <Contributors usernames="andreafalzetti,felladrin" />
- [#13161](https://github.com/gitpod-io/gitpod/pull/13161) - Admin dashboard now has pagination <Contributors usernames="easyCZ,geropl,svenefftinge" />
- [#13108](https://github.com/gitpod-io/gitpod/pull/13108) - Fix reading .gitpod.yml for self-managed GHE instances. <Contributors usernames="AlexTugarev,geropl" />
- [#13065](https://github.com/gitpod-io/gitpod/pull/13065) - Added a usage view that displays past workspace sessions on individual and team accounts. <Contributors usernames="easyCZ,jldec,svenefftinge" />
- [#13084](https://github.com/gitpod-io/gitpod/pull/13084) - Update GoLand IDE image to version 222.4167.25. <Contributors usernames="felladrin" />
- [#13057](https://github.com/gitpod-io/gitpod/pull/13057) - improved automated code configuration service for `go` <Contributors usernames="Siddhant-K-code,svenefftinge" />
- [#13001](https://github.com/gitpod-io/gitpod/pull/13001) - Update GitLab API library, which fixes paginated API requests. <Contributors usernames="AlexTugarev,gtsiolis,metcalfc,svenefftinge" />
- [#13033](https://github.com/gitpod-io/gitpod/pull/13033) - Update PhpStorm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#13032](https://github.com/gitpod-io/gitpod/pull/13032) - Update PyCharm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#12670](https://github.com/gitpod-io/gitpod/pull/12670) - Dismiss Usage Limit notifications automatically. <Contributors usernames="AlexTugarev,geropl,jldec,svenefftinge" />
- [#12994](https://github.com/gitpod-io/gitpod/pull/12994) - Update IntelliJ IDEA IDE image to version 222.4167.29. <Contributors usernames="felladrin" />
- [#12412](https://github.com/gitpod-io/gitpod/pull/12412) - Add beta notice and label on usage <Contributors usernames="easyCZ,filiptronicek,gtsiolis,laushinka" />
- [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
- [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
- [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
- [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12669](https://github.com/gitpod-io/gitpod/pull/12669) - Reword "Spending Limit" to "Usage Limit" <Contributors usernames="AlexTugarev,andrew-farries,jldec" />
- [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace <Contributors usernames="akosyakov,andreafalzetti,felladrin" />

```

Comment (loujaybee) [reply src/lib/contents/changelog/2022-09-30.md:32]: Update based on: https://github.com/gitpod-io/website/pull/2759/files adjusted script

**Comment by akosyakov** [comment]
#### JetBrains:
  - displaying information about workspace classes in the backend control center, separating infor about workpace from node
    - we could highlight what is the difference
  - standard actions in the backend control center, like extend timeout, stop workspace, docs, and so on

#### VS Code:
  - New ports panel now is default in VS Code Browser
  - VS Code Desktop is approaching GA, support of installing local extensions, fixed port auto detection and expose ports view aligned with new ports panel in VS Code Browser
    - it depends on landing https://github.com/gitpod-io/openvscode-server/pull/440 before Changelog, but after SH release cc @jeanp413 
    - should be highlight usability study?
  - SSH Gateway is default now for all users in VS Code Desktop, ping SH installations to enable SSH Gateway

**Comment by loujaybee** [comment]
😍 - Thanks so much @akosyakov, I've updated the PR description with your suggestions (to track)

> displaying information about workspace classes in the backend control center, separating infor about workpace from node - @akosyakov 

Added information here. Also mentioned the workspace classes work, and would be good to get @atduarte's feedback on it. 

> VS Code Desktop is approaching GA, support of installing local extensions, fixed port auto detection and expose ports view aligned with new ports panel in VS Code Browser it depends on landing https://github.com/gitpod-io/openvscode-server/pull/440 - @akosyakov 

Let's see how @jeanp413 gets on, we've got a lot for this changelog. We could do a push on VS Code Desktop GA next month? 

> should be highlight usability study? - @akosyakov 

🤔 - Not sure, I'm err'ing to no, but I'll think it! 

> SSH Gateway is default now for all users in VS Code Desktop, ping SH installations to enable SSH Gateway - @akosyakov 

Hrm, this might make more sense in the Self-Hosted release, I'll ask @lucasvaltl about this 🙏

File: src/lib/contents/changelog/2022-09-30.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+- [JetBrains Backend Control Center - Common Actions](#jetbrains-backend-control-center---common-actions)
+- [JetBrains Backend Control Center - Workspace Classes](#jetbrains-backend-control-center---workspace-classes)
+- [JetBrains Reconnect With Open Windows](#jetbrains-reconnect-with-open-windows)
+- [New Ports View in VS Code](#new-ports-view-in-vs-code)
+- [SSH Gateway default for Self-Hosted VS Code Desktop Users](#ssh-gateway-default-for-self-hosted-vs-code-desktop-users)
+- [Fixes and improvements](#fixes-and-improvements)
+
+<!-- TODO: Any insiders notes to highlight? -->
+
+### JetBrains Backend Control Center - Common Actions
```
Comment (akosyakov) [inline src/lib/contents/changelog/2022-09-30.md:27]: I wonder whether we could find better name than `Common`? Maybe `Gitpod Workspace Actions`?

**Review by akosyakov — APPROVED** [review]
LGTM from IDE side

There seem to be some improvements from webapp side in commits section at the bottom. Maybe we could collaborate to help adding them as well?

File: src/lib/contents/changelog/2022-09-30.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+All from the comfort of your IDE.
+
+### JetBrains Backend Control Center - Workspace Classes
+
+<!-- TODO: Add more zoomed in image that "points to the class information -->
+<!-- TODO: Add something about node vs workspace performance -->
+
+![](https://user-images.githubusercontent.com/2318450/188240455-e5cfba19-5c65-493a-b8c0-0e63c34fd17b.png)
+
+Did you know that when you're working with JetBrains IDEs in Gitpod you can [see real-time the performance of your workspace](/docs/ides-and-editors/intellij#workspace-performance) via the JetBrains Backend Control Center? This feature is useful when your workspace isn't performing as you'd expect and you want to gather more information. In addition to our recent additions of `Workspace CPU` and `Workspace Memory` we've now added the actual workspace class to the information shown in the control center.
```
Comment (atduarte) [inline src/lib/contents/changelog/2022-09-30.md:74]:
```suggestion
Did you know that when you're working with JetBrains IDEs in Gitpod you can [monitor the performance of your workspace in real-time](/docs/ides-and-editors/intellij#workspace-performance) via the JetBrains Backend Control Center? This feature is useful when your workspace isn't performing as you'd expect and you want to gather more information. In addition to our recent additions of `Workspace CPU` and `Workspace Memory` we've now added the actual workspace class to the information shown in the control center.
```

File: src/lib/contents/changelog/2022-09-30.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+... but why? When you work in a cloud developer environment, you should have the flexibility to choose the right performance profile for your project. Recently we've been working to bring to Gitpod a feature we call "Workspaces Classes" [#8261](https://github.com/gitpod-io/gitpod/issues/8261). With Workspaces Classes you'll be able to customize and configure the specification of your workspace. We know that for larger projects, especially large Java projects using IntelliJ require more resources than the current Gitpod workspaces are designed for.
```
Comment (atduarte) [inline src/lib/contents/changelog/2022-09-30.md:76]:
```suggestion
... but why? When you work in a cloud developer environment, you should have the flexibility to choose the right performance profile for your project. Recently we've been working to bring to Gitpod a feature we call "Workspaces Classes" [#8261](https://github.com/gitpod-io/gitpod/issues/8261). With Workspaces Classes you'll be able to select workspace specifications that better match your use case. For example, you will be able to get more powerful workspaces to run larger Java projects and IntelliJ.
```

File: src/lib/contents/changelog/2022-09-30.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+We introduced the new ports view as we wanted to align our experiences on VS Code Desktop with VS Code Browser. We know lots of users like to use Gitpod in the browser sometimes for "smaller changes" like reviewing pull requests. But for more "deep work" many users opt for VS Code Desktop. However and wherever you choose to code, Gitpod will make that as much a comfortable and familiar experience as you're used to.
+
+### SSH Gateway default for Self-Hosted VS Code Desktop Users
+
+<!-- TODO: Add content -->
+
+### Fixes and improvements
+
+- [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />
```
Comment (atduarte) [inline src/lib/contents/changelog/2022-09-30.md:104]: @loujaybee order of contributors seems borked 🤔 Not problematic, but something to note

Comment (filiptronicek) [reply src/lib/contents/changelog/2022-09-30.md:104]: @atduarte from what I saw it's just ordered alphabetically, I think this is fine as long as the actual PR author is the first one unrelated to their alphabetical order in the list, will incorporate that into https://github.com/gitpod-io/website/pull/2763 🧡.

File: src/lib/contents/changelog/2022-09-30.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+- [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
+- [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
+- [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
+- [#12669](https://github.com/gitpod-io/gitpod/pull/12669) - Reword "Spending Limit" to "Usage Limit" <Contributors usernames="AlexTugarev,andrew-farries,jldec" />
+- [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
+- [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
```
Comment (filiptronicek) [inline src/lib/contents/changelog/2022-09-30.md:124]: Here are the latest updates (cc @loujaybee):
```suggestion
- [#13322](https://github.com/gitpod-io/gitpod/pull/13322) - Fix issue that sporadically prevented workspaces to be opened from a GitHub issue <Contributors usernames="aledbf,atduarte,jenting,utam0k" />
- [#13144](https://github.com/gitpod-io/gitpod/pull/13144) - Enable configuration of core dump behavior in a workspace <Contributors usernames="Furisto,aledbf,csweichel,david-bakin,svenefftinge,utam0k" />
- [#13053](https://github.com/gitpod-io/gitpod/pull/13053) - Do not check all existing git branches during the initial checkout <Contributors usernames="aledbf,jenting,kylos101,sagor999,utam0k" />
- [#13205](https://github.com/gitpod-io/gitpod/pull/13205) - JetBrains: VM options to install plugins <Contributors usernames="akosyakov,andreafalzetti,filiptronicek,subinamathew" />
- [#12445](https://github.com/gitpod-io/gitpod/pull/12445) - Support for storing VS Code edit sessions in sync server <Contributors usernames="akosyakov,andrew-farries,filiptronicek,jeanp413,mustard-mh" />
- [#13295](https://github.com/gitpod-io/gitpod/pull/13295) - Avoid second prebuild been triggered on same commit. <Contributors usernames="AlexTugarev,geropl" />
- [#13296](https://github.com/gitpod-io/gitpod/pull/13296) - Fail prebuild if image build fails. <Contributors usernames="AlexTugarev,easyCZ" />
- [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />
- [#13203](https://github.com/gitpod-io/gitpod/pull/13203) - [server]: Add a liveness probe which fails when the nodejs event loop lag exceeds a certain threshold <Contributors usernames="andrew-farries,easyCZ" />
- [#13080](https://github.com/gitpod-io/gitpod/pull/13080) - JetBrains Gateway: Avoid 30 seconds delay when connecting to a workspace using an expired token <Contributors usernames="andreafalzetti,felladrin" />
- [#13161](https://github.com/gitpod-io/gitpod/pull/13161) - Admin dashboard now has pagination <Contributors usernames="easyCZ,geropl,svenefftinge" />
- [#13108](https://github.com/gitpod-io/gitpod/pull/13108) - Fix reading .gitpod.yml for self-managed GHE instances. <Contributors usernames="AlexTugarev,geropl" />
- [#13065](https://github.com/gitpod-io/gitpod/pull/13065) - Added a usage view that displays past workspace sessions on individual and team accounts. <Contributors usernames="easyCZ,jldec,svenefftinge" />
- [#13084](https://github.com/gitpod-io/gitpod/pull/13084) - Update GoLand IDE image to version 222.4167.25. <Contributors usernames="felladrin" />
- [#13057](https://github.com/gitpod-io/gitpod/pull/13057) - improved automated code configuration service for `go` <Contributors usernames="Siddhant-K-code,svenefftinge" />
- [#13001](https://github.com/gitpod-io/gitpod/pull/13001) - Update GitLab API library, which fixes paginated API requests. <Contributors usernames="AlexTugarev,gtsiolis,metcalfc,svenefftinge" />
- [#13033](https://github.com/gitpod-io/gitpod/pull/13033) - Update PhpStorm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#13032](https://github.com/gitpod-io/gitpod/pull/13032) - Update PyCharm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#12670](https://github.com/gitpod-io/gitpod/pull/12670) - Dismiss Usage Limit notifications automatically. <Contributors usernames="AlexTugarev,geropl,jldec,svenefftinge" />
- [#12994](https://github.com/gitpod-io/gitpod/pull/12994) - Update IntelliJ IDEA IDE image to version 222.4167.29. <Contributors usernames="felladrin" />
- [#12412](https://github.com/gitpod-io/gitpod/pull/12412) - Add beta notice and label on usage <Contributors usernames="easyCZ,filiptronicek,gtsiolis,laushinka" />
- [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
- [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
- [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
- [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12669](https://github.com/gitpod-io/gitpod/pull/12669) - Reword "Spending Limit" to "Usage Limit" <Contributors usernames="AlexTugarev,andrew-farries,jldec" />
- [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
```

P.S.: I removed the long release note from https://github.com/gitpod-io/gitpod/pull/13205 - I think before we figure out the multiline changelog entries, we should add these manually to their manually curated subsections.

**Review by jldec — APPROVED** [review]
LGTM thanks. 🙏🏽

**Comment by atduarte** [comment]
Fixes and improvements list needs to be regenerated. Not sure I can run it myself 🤔

**Comment by filiptronicek** [comment]
> Fixes and improvements list needs to be regenerated. Not sure I can run it myself 🤔

Just done that, updated the list in my code suggestion from yesterday,.

**Comment by akosyakov** [comment]
@loujaybee @lucasvaltl for air-gapped SH this one is important: https://github.com/gitpod-io/gitpod/pull/13205 Maybe better for SH changelog.

**Review by atduarte — CHANGES_REQUESTED** [review]
Image should contain new key highlights introduced yesterday:
- Configurable core dump behavior
- Faster startup times

Also there are a couple of TODOs left.

**Comment by loujaybee** [comment]
> Image should contain new key highlights introduced yesterday:

Which image are you referring to here, @atduarte ? Not sure I follow. 

> Also there are a couple of TODOs left.

Looking now!

**Comment by atduarte** [comment]
> Which image are you referring to here, @atduarte ? Not sure I follow.
@loujaybee Oops, forgot to include the reference: `static/images/changelog/2022-09-30/open-graph.jpg`

**Comment by jacobparis** [comment]
> > Which image are you referring to here, @atduarte ? Not sure I follow.
> > @loujaybee Oops, forgot to include the reference: `static/images/changelog/2022-09-30/open-graph.jpg`

I will take this

File: src/lib/contents/changelog/2022-09-30.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+- [JetBrains Backend Control Center - Gitpod workspace actions](#jetbrains-backend-control-center---gitpod-workspace-actions)
+- [JetBrains Backend Control Center - Workspace Classes](#jetbrains-backend-control-center---workspace-classes)
+- [JetBrains Reconnect With Open Windows](#jetbrains-reconnect-with-open-windows)
+- [New Ports View in VS Code](#new-ports-view-in-vs-code)
+- [SSH Gateway default for Self-Hosted VS Code Desktop Users](#ssh-gateway-default-for-self-hosted-vs-code-desktop-users)
+- [Configurable core dump behavior](#configurable-core-dump-behavior)
+- [Faster startup times for large repositories](#faster-startup-times-for-large-repositories)
+- [Fixes and improvements](#fixes-and-improvements)
+
+<!-- TODO: Any insiders notes to highlight? -->
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-09-30.md:28]:
```suggestion
```

**Comment by jacobparis** [comment]
There were some broken links to the docs that were breaking compilation (redirects were set but compile doesn't fix), I've pushed a commit to fix those

**Comment by Cahllagerfeld** [comment]
There's still a todo for updating the regular image @loujaybee. Is this still relevant? If yes, do you have something particular in mind already?

**Comment by jacobparis** [comment]
Is this the final list of changes to include in the social share image?

image

**Comment by jacobparis** [comment]
why is CMD+Enter close and comment, that's never what I want

**Comment by loujaybee** [comment]
- Moved all images in repo
- Updated alt descriptions
- Added image captions

File: src/lib/contents/changelog/2022-09-30.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
+---
+title: September Gitpod Release 2022
+excerpt:
+date: 2022-09-30
+# TODO: Update image
+image: 2022-09-30.png
+ogImage: 2022-09-30/open-graph.jpg
+alt:
+---
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-09-30.md:9]: We should delete the old copied image: `static/images/changelog/2022-09-30.png` also. 

This open graph image is not showing for me? @jacobparis 

```suggestion
---
title: September Gitpod Release 2022
excerpt:
date: 2022-09-30
image: 2022-09-30/open-graph.jpg
alt:
---
```

Comment (Cahllagerfeld) [reply src/lib/contents/changelog/2022-09-30.md:9]: It's not on gitpod.io yet. There's a bug with sveltest that we can't display it dynamically. I tested it in my preview though.

**Comment by loujaybee** [comment]
> Is this the final list of changes to include in the social share image? - @jacobparis 

I think needs to be updated, as we updated the title so it's no longer "common actions". 

Can we resync with the table of contents?

File: src/lib/contents/changelog/2022-09-30.md (hunk 8 above)
Comment (Cahllagerfeld) [reply src/lib/contents/changelog/2022-09-30.md:9]: Do we want to put the OpenGraph-Image (or the same style of image) as the header image?

File: src/lib/contents/changelog/2022-09-30.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+title: September Gitpod Release 2022
+excerpt:
+date: 2022-09-30
+# TODO: Update image
+image: 2022-09-30.png
```
Comment (atduarte) [inline src/lib/contents/changelog/2022-09-30.md:6]: Colors seem muted. Particularly the Gitpod logo. Is this on purpose?

Comment (loujaybee) [reply src/lib/contents/changelog/2022-09-30.md:6]: I'm a little confused, I thought we would swap for [this image](https://github.com/gitpod-io/website/pull/2760#issuecomment-1263531785). The one here is just a placeholder copied from the previous changelog earlier on so the build would pass. 

See also: https://github.com/gitpod-io/website/pull/2760#discussion_r984565307

File: src/lib/contents/changelog/2022-09-30.md (hunk 8 above)
Comment (jacobparis) [reply src/lib/contents/changelog/2022-09-30.md:9]: No the open graph image is distinct from the header image

Comment (jacobparis) [reply src/lib/contents/changelog/2022-09-30.md]: > There's a bug with Sveltekit that we can't display it dynamically. I tested it in my preview though.

Not a bug, we just can't statically generate pages that include URLs before we know what URL we're going to be deploying it to. Serverside rendering is ideal here

**Review by jacobparis — APPROVED** [review]

**Comment by atduarte** [comment]
image

Nit: Where can we add this description, like we had in August?

**Review by atduarte — APPROVED** [review]

**Comment by Cahllagerfeld** [comment]
Let me check @atduarte

File: src/lib/contents/changelog/2022-09-30.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
+---
+title: September Gitpod Release 2022
+excerpt:
```
Comment (Siddhant-K-code) [inline src/lib/contents/changelog/2022-09-30.md:3]: Fill `excerpt` to resolve [this](https://github.com/gitpod-io/website/pull/2760#issuecomment-1263723240)

File: src/lib/contents/changelog/2022-09-30.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
+---
+title: September Gitpod Release 2022
+excerpt: The latest Gitpod updates for September 2022, including improvements for JetBrains, improving the startup times for large repos and lots of other fixes and improvements.
```
Comment (Cahllagerfeld) [inline src/lib/contents/changelog/2022-09-30.md:3]: I added an excerpt here @atduarte, feel free to adjust it!

Comment (atduarte) [reply src/lib/contents/changelog/2022-09-30.md:3]: Thank you, @Cahllagerfeld! Looks great

**Comment by loujaybee** [comment]
If I could approve my own PR, I would ☑️

Good hustle, team! 🚀

**Pull request merged**